configs/stdc/stdc-1446_pretrain_512x1024_80k_cityscapes.py

```diff
@@ -0,0 +1,6 @@
+_base_ = './stdc-1446_512x1024_80k_cityscapes.py'
```

Comment:
Please use the correct `pretrain` name according to pretrained dataset (in1k-pre or something else).